ospd/protocol.py

```diff
@@ -146,6 +146,32 @@ def process_credentials_elements(cred_tree: Element) -> Dict:
 
         return credentials
 
+    @staticmethod
+    def process_alive_test_methods(cred_tree: Element, options: Dict) -> None:
```

Comment:
Maybe a more descriptive param name.
```suggestion
    def process_alive_test_methods(alive_test_tree: Element, options: Dict) -> None:
```